geomesa-cassandra/geomesa-cassandra-datastore/src/test/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreTest.scala

```diff
@@ -236,7 +236,7 @@ object CassandraDataStoreTest {
 
       System.setProperty("cassandra.storagedir", storagedir.getPath)
 
-      EmbeddedCassandraServerHelper.startEmbeddedCassandra("cassandra-config.yaml", 30000L)
+      EmbeddedCassandraServerHelper.startEmbeddedCassandra("cassandra-config.yaml", 1200000L)
```

Comment:
@elahrvivaz @jahhulbert-ccri I'm for merging this and seeing if it helps any of the nightly build failures at LT's Hudson or Travis CI builds.

Thoughts?